lib/cfn-nag/cfn_nag_executor.rb

```diff
@@ -69,7 +69,7 @@ def scan_file(cfn_nag, fail_on_warnings)
   end
 
   def validate_options(opts)
-    unless opts[:output_format].nil? || %w[txt json].include?(opts[:output_format])
+    unless opts[:output_format].nil? || %w[colortxt txt json].include?(opts[:output_format])
       Trollop.die(:output_format,
                   'Must be txt or json')
```

Comment:
shouldn't this include colortxt now?

Reply:
yea the die message should include colortxt